src/brasil/gov/portal/browser/plone/templates/listing_summary.pt

```diff
@@ -57,6 +75,7 @@
               tags:
               <span tal:repeat="tag tags">
                   <a href=""
+                     rel="nofollow"
```

Comment:
I don't know if you can have 2 `rel` attributes; maybe you should change it for `rel="nofollow tag"`; try to validate it.